scripts/check-live-demo.sh

```
@@ -0,0 +1,211 @@
+        code="$(fetch "$BASE$start_url")"
+        [ "$code" = "200" ] \
+            && pass "start_url resolves" \
+            || fail "start_url $start_url returns $code, so the installed app opens on an error"
+    fi
+fi
+
+# ---------------------------------------------------------------- the rest of the surface
+printf '\nEverything else the demo has to serve\n'
+for path in /baryodev-pwa.js /umbraco /build-info /icon-192.png; do
+    code="$(fetch "$BASE$path")"
+    [ "$code" = "200" ] && pass "$path is 200" || fail "$path returned $code"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)